src/users/users.controller.ts

```diff
@@ -283,6 +285,42 @@ export class UsersController {
     }
   }
 
+  @Get('/one/:email/isadmin')
+  @UseGuards(AuthGuard)
+  @JwtType(JwtProcessorType.RSA)
+  @ApiOperation({
+    description: SWAGGER_DESC_ADMIN_RIGHTS,
+  })
+  @ApiForbiddenResponse({
+    description: 'user has no admin rights',
+    schema: {
+      type: 'object',
+      properties: {
+        statusCode: { type: 'number' },
+        message: { type: 'string' },
+      },
+    },
+  })
+  @ApiOkResponse({
+    description: 'Returns true if user has admin rights',
+  })
+  async getAdminStatus(@Param('email') email: string) {
+    try {
+      return await this.usersService.findByEmail(email).then(({ isAdmin }) => {
+        if (isAdmin === true) {
+          return isAdmin;
```

Comment:
Please define a view model. e.g.

```ts
class PermissionsDto {
  public admin!: boolean;
}
```

Reply:
We have already defined property 'isAdmin' in user model. Do you think we need additional model for this?